models/product/src/product/product-draft/presets/sample-data-fashion/necklace.spec.ts

```diff
@@ -0,0 +1,354 @@
+import { TProductDraft } from '../../../types';
+import necklace from './necklace';
+
+describe(`with necklace preset`, () => {
+  it('should return a sample necklace product preset', () => {
+    const necklacePreset = necklace().build<TProductDraft>();
+    expect(necklacePreset).toMatchInlineSnapshot(`
+      {
+        "categories": undefined,
+        "categoryOrderHints": undefined,
+        "description": undefined,
+        "key": "necklace",
+        "masterVariant": {
+          "assets": [],
+          "attributes": [
+            {
+              "name": "type",
+              "value": {
+                "key": "Jewelry",
+                "label": "Jewelry",
+              },
+            },
+            {
+              "name": "engraving",
+              "value": "Happy Anniversary",
+            },
+          ],
+          "images": [
+            {
+              "dimensions": {
+                "h": 122,
+                "w": 103,
```

Comment:
The [source file](https://github.com/commercetools/test-data-research/blob/main/src/entity-drafts/products.ts#LL45C12-L45C12) has a different h and w, not sure if relevant

Reply:
It looks like they build in a different order, but that the values are correct?

Reply:
Ah, thanks. You're right, I got switched up